variants/audit-logging/lib/auditor_log_formatter.rb

```diff
@@ -0,0 +1,16 @@
+require "logger"
+##
+# We want everything which is sent to `Rails.logger` to be parsable as JSON.
+# This means we need to remove the Timestamp and severity prefix from the log
+# lines that Rails normally generates. We do this with this customer Log
+# formatter - see `config/application.rb` for where Rails is instructed to use
+# it.
+#
+class AuditorLogFormatter < Logger::Formatter
+  ##
+  # The public API of this function is defined by `Logger::Formatter`
+  #
+  def call(_severity, _timestamp, _progname, msg)
+    "#{msg}\n"
+  end
+end
```

Comment:
This file is loaded before Rails loads `app/services` so we cannot keep the formatter under `app/services`. I have put it in `lib/` because that seems the most idiomatic for this kind of usage.